docs/products/opensearch/howto/opensearch-dashboard-multi_tenancy.rst

```diff
@@ -0,0 +1,76 @@
+Set up OpenSearch® Dashboard multi-tenancy |beta|
+==================================================
+
+Aiven for OpenSearch® provides support for multi-tenancy through OpenSearch Security Dashboard. Multi-tenancy in OpenSearch Security enables multiple users or groups to securely access the same OpenSearch cluster while maintaining their distinct permissions and data access levels. With multi-tenancy, each tenant has its own isolated space for working with indexes, visualizations, dashboards, and other OpenSearch objects, ensuring tenant-specific data and resources are protected from unauthorized access. 
+
+Prerequisites
+-------------
+
+* Aiven for OpenSearch 
+* Administrative access to both the Aiven for OpenSearch service and OpenSearch Dashboard
+
+Configure multi-tenancy in OpenSearch® Dashboard
+-------------------------------------------------
+
+This section provides information on configuring multi-tenancy in OpenSearch Dashboard, which involves enabling OpenSearch Security management, creating tenants, assigning roles, and mapping roles to users.
+
+Step 1: Enable OpenSearch® Security management
+```````````````````````````````````````````````
+The first step in setting up multi-tenancy in OpenSearch Dashboard is to enable OpenSearch Security management on your Aiven for OpenSearch service. Security provides authentication and authorization features that are required for multi-tenancy.  
+To enable OpenSearch Security management, see docs <link pending update>. 
```

Comment:
pending link update

Reply:
Yeah, that is a placeholder, as I cannot link the "Enable OpenSearch Security management" topic since it is not merged. The build will fail due to a missing link.